Six new Heliarch ships (#5484)
endless-sky/endless-sky: data/coalition/coalition ships.txt

	thumbnail "thumbnail/heliarch breacher"
	attributes
		category "Light Warship"
		licenses
			Heliarch
		"cost" 17659000
		"shields" 18800
		"hull" 16500
		"required crew" 11
		"bunks" 15
		"mass" 200
		"drag" 3.2
		"heat dissipation" .77
		"ion protection" .1
		"fuel capacity" 800
		"cargo space" 13
		"outfit space" 372
		"weapon capacity" 112
		"engine capacity" 106
		weapon
			"blast radius" 150
			"shield damage" 1200
			"hull damage" 600
			"hit force" 1500
	outfits
		"Bombardment Cannon" 2
		"Bombardment Turret"
		"Finisher Pod"
		"Finisher Torpedo" 40
		
		"Small Reactor Module" 2
		"Small Battery Module" 2
		"Overcharged Shield Module"
		"Overclocked Repair Module"
		"Cooling Module" 2
		"Scanning Module"
		"Enforcer Riot Staff" 8
		"Enforcer Confrontation Gear" 6
		
		"Large Thrust Module"
		"Small Thrust Module" 3
		"Small Steering Module"
		"Hyperdrive"
	
	engine 0 85 .9
	engine -19 78 .7
	engine 19 79 .7
	gun 0 -87 "Finisher Pod"
	gun -20 -33 "Bombardment Cannon"
	gun 20 -33 "Bombardment Cannon"
	turret 0 -23.5 "Bombardment Turret"
	explode "small explosion" 30
	explode "medium explosion" 40
	explode "large explosion" 20
	"final explode" "final explosion medium" 1

> 	description `The Breacher isn't the most durable Heliarch ship, but its impressive forward speed and high firepower for a ship of this size make it ideal for strafing runs.`

Reviewer: ```suggestion
	description `The Breacher isn't the most durable Heliarch vessel, but its impressive forward speed and high firepower for its size make this ship ideal for strafing runs.`
```
Just style, so optional.

		"shields" 54900
		"hull" 45300
		"required crew" 26
		"bunks" 41
		"mass" 420
		"drag" 5.3
		"heat dissipation" .71
		"ion protection" .2
		"fuel capacity" 1500
		"cargo space" 27
		"outfit space" 552
		"weapon capacity" 201
		"engine capacity" 141
		weapon
			"blast radius" 200
			"shield damage" 2700
			"hull damage" 1200
			"hit force" 3000
	outfits
		"Finisher Pod" 3
		"Finisher Torpedo" 120
		"Bombardment Turret"
		"Heliarch Attractor"
		
		"Large Reactor Module"
		"Large Battery Module"
		"Overcharged Shield Module"
		"Overclocked Repair Module"
		"Cooling Module" 2
		"Scanning Module" 2
		"Enforcer Riot Staff" 25
		"Enforcer Confrontation Gear" 13
		
		"Large Thrust Module" 2
		"Small Thrust Module"
		"Large Steering Module"
		"Small Steering Module" 2
		"Scram Drive"
	
	engine -5.5 103 .1
	engine 5.5 103 .1
	engine -25 113 .8
	engine 25 113 .8
	gun 0 -117 "Finisher Pod"
	gun -17.5 -72.5 "Finisher Pod"
	gun 17.5 -72.5 "Finisher Pod"
	turret 0 -39.5 "Heliarch Attractor"
	turret 0 33 "Bombardment Turret"
	explode "small explosion" 30
	explode "medium explosion" 50
	explode "large explosion" 25
	explode "huge explosion" 5
	"final explode" "final explosion large" 1

> 	description `Even if they mostly serve as patrol vessels due to the lack of conflict in Coalition space, Hunters were originally designed to quickly track down dissidents and bring them to justice.`

Reviewer: ```suggestion
	description `Although they mostly serve as patrol vessels nowadays in peacetime, Hunters were originally designed to quickly track down dissidents and bring them to justice.`
```
Again, optional.

		"mass" 570
		"drag" 7.9
		"heat dissipation" .67
		"ion protection" .2
		"fuel capacity" 900
		"cargo space" 136
		"outfit space" 524
		"weapon capacity" 164
		"engine capacity" 114
		weapon
			"blast radius" 250
			"shield damage" 2800
			"hull damage" 1400
			"hit force" 3800
	outfits
		"Finisher Pod" 2
		"Finisher Torpedo" 120
		"Finisher Storage Tube" 2
		"Ion Hail Turret"
		
		"Large Reactor Module"
		"Small Battery Module"
		"Large Battery Module"
		"Overcharged Shield Module"
		"Overclocked Repair Module" 2
		"Cooling Module" 2
		"Scanning Module" 4
		"Enforcer Riot Staff" 83
		"Enforcer Confrontation Gear" 49
		
		"Small Thrust Module" 2
		"Small Steering Module"
		"Large Thrust Module" 2
		"Large Steering Module"
		"Scram Drive"
	
	engine -33 108 .5
	engine -17 163 .7
	engine 17 163 .7
	engine 33 108 .5
	gun -13 -124.5 "Finisher Pod"
	gun 13 -124.5 "Finisher Pod"
	turret 0 45 "Ion Hail Turret"
	fighter -34.5 -82
	fighter 34.5 -82
	fighter -67.5 35
	fighter -67.5 35
	fighter 67.5 35
	fighter 67.5 35
	explode "small explosion" 20
	explode "medium explosion" 50
	explode "large explosion" 20
	explode "huge explosion" 10
	"final explode" "final explosion large" 1

> 	description `During their time battling the Quarg, the species of the Coalition focused on producing the strongest, toughest warships they could, so fightercraft were out of question. Now, however, the Heliarch's shield matrix and hull integrity technology is much more advanced, and they have recently experimented with such ships. The Judicator was made for transportation of these fightercraft.`

Reviewer 2: ```suggestion
	description `During their time battling the Quarg, the Coalition focused on producing the strongest, toughest warships they could, so fightercraft were out of question. Now, however, the Heliarch's shield matrix and hull integrity technology is much more advanced, and they have recently experimented with such ships. The Judicator was made for transportation of these new fighters.`
```

> 	description `During their time battling the Quarg, the species of the Coalition focused on producing the strongest, toughest warships they could, so fightercraft was something out of question. Now however, the Heliarch's shield matrix and hull integrity technology is much more advanced, and they have recently started experimenting with such ships, and out of a need for something to carry them, the Judicator was made.`

Reviewer: ```suggestion
	description `During their time battling the Quarg, the species of the Coalition focused on producing the strongest, toughest warships they could, so fightercraft were out of question. Now, however, the Heliarch's shield matrix and hull integrity technology is much more advanced, and they have recently started experimenting with such ships. Out of a need for something to carry them, the Judicator was made.`
```
OR
```suggestion
	description `During their time battling the Quarg, the species of the Coalition focused on producing the strongest, toughest warships they could, so fightercraft were out of question. Now, however, the Heliarch's shield matrix and hull integrity technology is much more advanced, and they have recently experimented with such ships. The Judicator was made for transportation of these fightercraft.`
```
First is purely corrections, second is suggested stylistic changes (optional).

> 	description `	Alongside their six fighter bays, Judicators come with a large bunk count and higher cargo space than other Heliarch warships, doubling as a logistics vessel for transporting troops and equipment.`

Reviewer: ```suggestion
	description `	Alongside their six fighter bays, Judicators come with a large bunk count and higher cargo space than other Heliarch warships, thus doubling as a logistics vessel for transporting troops and equipment.`
```
`thus` just helps the train of thought travel more smoothly to me. Optional.

		"Hyperdrive"

ship "Heliarch Pursuer"
	sprite "ship/heliarch pursuer"
	thumbnail "thumbnail/heliarch pursuer"
	attributes
		category "Fighter"
		licenses
			Heliarch
		"cost" 1029000
		"shields" 2200
		"hull" 1600
		"required crew" 1
		"bunks" 1
		"mass" 31
		"drag" 0.8
		"heat dissipation" 0.99
		"outfit space" 105
		"weapon capacity" 19
		"engine capacity" 36
		weapon
			"blast radius" 15
			"shield damage" 150
			"hull damage" 200
			"hit force" 350
	outfits
		"Ion Rain Gun"
		
		"Large Collector Module"
		"Small Collector Module"
		"Small Battery Module" 3
		"Small Shield Module"
		"Small Repair Module"
		
		"Small Steering Module" 2
		"Small Thrust Module" 2
	
	engine 0 34.5
	engine 7 33.5
	engine -7 33.5
	gun 0 -35.5
	explode "tiny explosion" 20
	explode "small explosion" 10

> 	description `Occasionally, when a Heliarch cadet shows promising results in dogfighting simulations, they are allowed to fly a ship of their own, that being the Pursuer.`

Reviewer: ```suggestion
	description `Occasionally, when a Heliarch cadet shows promising results in dogfighting simulations, they are allowed to fly a ship of their own: the Pursuer.`
```
Iffy about the grammatical correctness of the sentence as-is. Could be right; I just don't know. I tried to suggest something that I know is right and still kept the rhythm of your original sentence.

> 	description `	Though certainly not as durable as the other fighters in Coalition space, this Kimek design is cheaper to produce, and is capable of outrunning any other Coalition ship.`

Reviewer: ```suggestion
	description `	Though certainly not as durable as the other fighters in Coalition space, this Kimek design is cheaper to produce and capable of outrunning any other Coalition ship.`
```

ship "Heliarch Pursuer" "Heliarch Pursuer (Scrappy)"
	outfits
		"Bombardment Cannon"
		
		"Large Collector Module"
		"Small Collector Module" 3
		"Small Battery Module"
		"Small Shield Module"
		"Small Repair Module"
		
		"Small Thrust Module" 2
		"Small Steering Module" 2

ship "Heliarch Rover"
	sprite "ship/heliarch rover"
	thumbnail "thumbnail/heliarch rover"
	attributes
		category "Fighter"
		licenses
			Heliarch
		"cost" 1238000
		"shields" 3300
		"hull" 3100
		"required crew" 1
		"bunks" 1
		"mass" 37
		"drag" 1.19
		"heat dissipation" .96
		"outfit space" 125
		"weapon capacity" 24
		"engine capacity" 32
		weapon
			"blast radius" 15
			"shield damage" 150
			"hull damage" 200
			"hit force" 350
	outfits
		"Bombardment Cannon" 2
		
		"Large Collector Module"
		"Large Battery Module"
		"Small Shield Module"
		"Small Repair Module" 3
		
		"Small Thrust Module" 2
		"Small Steering Module" 2
	
	engine 7 28
	engine -7 28
	gun 5 -30
	gun -5 -30
	explode "tiny explosion" 25
	explode "small explosion" 15

> 	description `The very first Fighter the Heliarchs designed, Rovers are well-rounded ships that the Heliarchs employ regularly due to their reliability in any situation.`

Reviewer: ```suggestion
	description `The very first fighter they designed, Rovers are well-rounded ships that the Heliarchs employ regularly due to their reliability in any situation.`
```
is `Fighter` supposed to be capitalized? I'm unsure. Also, double `the Heliarchs`.

ship "Heliarch Rover" "Heliarch Rover (Ions)"
	outfits
		"Ion Rain Gun"
		
		"Large Collector Module"
		"Small Collector Module" 3
		"Small Battery Module"
		"Small Shield Module"
		"Large Repair Module"
		
		"Small Thrust Module" 2
		"Small Steering Module" 2

ship "Heliarch Stalker"
	sprite "ship/heliarch stalker"
	thumbnail "thumbnail/heliarch stalker"
	attributes
		category "Fighter"
		licenses
			Heliarch
		"cost" 1358000
		"shields" 5700
		"hull" 5200
		"required crew" 1
		"bunks" 2
		"mass" 80
		"drag" 2.1
		"heat dissipation" 0.64
		"outfit space" 133
		"weapon capacity" 43
		"engine capacity" 16
		weapon
			"blast radius" 15
			"shield damage" 150
			"hull damage" 200
			"hit force" 350
	outfits
		"Ion Hail Turret"
		
		"Large Collector Module"
		"Small Collector Module"
		"Small Battery Module"
		"Large Shield Module"
		"Large Repair Module"
		
		"Small Steering Module"
		"Small Thrust Module"
	
	engine 0 40 .8
	turret 0 -1.5
	explode "tiny explosion" 25
	explode "small explosion" 15

> 	description `Boasting incredible durability for a ship this size, the Stalker is a compact fortress that can take a lot of punishment and recover faster than some warships. Its speed and maneuverability, however, are all but impressive.`

Reviewer: ```suggestion
	description `Boasting incredible durability for a ship this size, the Stalker is a compact fortress that can take serious punishment and recover faster than most other warships. Its speed and maneuverability, however, leave much to be desired.`
```
Just style.

Author: outside of human warships (that aren't using the S-series Regenerators), almost all other warships in the game have more regen than the Stalker's, so I left it at `some` instead of swapping for `most other`
